docs/develop/data-modeling.md

```diff
@@ -119,11 +116,23 @@ _asset scale_. For example, representing USD in cents uses an asset scale of `2`
 
 ### Oversized Amounts
 
-The other direction works as well. If the smallest useful unit of a currency is `10,000,000 ¤`, then
-it can be scaled down to the integer `1`.
+The other direction works as well. If the smallest useful unit of a currency is `10,000,000` units,
+then it can be scaled down to the integer `1`.
```

Comment:
https://en.wikipedia.org/wiki/Currency_sign_(generic) :stuck_out_tongue:

Reply:
I know! Still thought it was clearer to just say units 😁

Reply:
Yeah agreed, I just think its a fun symbol :smile: